flink-kubernetes-standalone/src/test/java/org/apache/flink/kubernetes/operator/kubeclient/decorators/CmdStandaloneJobManagerDecoratorTest.java

```diff
@@ -85,7 +86,8 @@ public void testApplicationCommandAndArgsAdded() {
                 containsInAnyOrder(
                         CmdStandaloneJobManagerDecorator.APPLICATION_MODE_ARG,
                         "--allowNonRestoredState",
-                        "false",
+                        "--fromSavepoint",
+                        "/tmp/savepoint/path",
```

Comment:
Why do we need to add this here? Seems like it worked before without it

Reply:
Because I'm testing the logic I added in the `CmdStandaloneJobManagerDecorator` file.